DEVELOPERS_GUIDE.md

```diff
@@ -0,0 +1,119 @@
+
+
+## Clojure & Leiningen
+
+### Clojure
+
+#### What is Clojure?
+Clojure is a dynamic development environment where you interact with your program while you write it, growing and adding to it while it’s running. To work with Clojure you need an editor that supports evaluation in source files and structural editing (working with nested forms in addition to character editing).
+
+#### Why we use it
+Clojure meets its goals by: embracing an industry-standard, open platform - the JVM; modernizing a venerable language - Lisp; fostering functional programming with immutable persistent data structures; and providing built-in concurrency support via software transactional memory and asynchronous agents. The result is robust, practical, and fast.
+
+### Leiningen
+
+#### What is Leiningen?
+Leiningen is a build automation and dependency management tool for the simple configuration of software projects written in the Clojure programming language.
+
+#### Why we use it
+Leiningen is the easiest way to use Clojure. With a focus on project automation and declarative configuration, it gets out of your way and lets you focus on your code.
+
+
+
+## Running the CMR
+The CMR is a system consisting of many services. The services can run
+individually or in a single process. Running in a single process makes
+local development easier because it avoids having to start many different
+processes. The `dev-system` project allows the CMR to run from a single REPL
+or JAR file. If you are developing a client against the CMR you can build and
+run the entire CMR with no external dependencies from this JAR file and use
+that instance for local testing. The sections below contain instructions for
+running the CMR as a single process or as many processes.
+
+### Using the `cmr` CLI Tool
+
+This project has its own tool that is able to do everything from initial setup to
+running builds and tests on the CI/CD infrastructure. To use the tool
+as we do below, be sure to run the following from the top-level CMR directory:
+
+```sh
+export PATH=$PATH:`pwd`/bin
+source resources/shell/cmr-bash-autocomplete
+```
+
+(If you use a system shell not compatible with Bash, we accept Pull Requests for
+new shells with auto-complete.)
+
+To make this change permanent:
+
+```sh
+echo "export PATH=\$PATH:`pwd`/bin" >> ~/.profile
+echo "source `pwd`/resources/shell/cmr-bash-autocomplete" >> ~/.profile
+```
+
+### Building and Running CMR Dev System in a REPL with CMR CLI tool
+
+1. `cmr setup profile` and then update the new `./dev-system/profiles.clj` file.
+   it will look something like this:
+   ``` clojure
+   {:dev-config {:env {:cmr-metadata-db-password "<YOUR PASSWORD HERE>"
+                       :cmr-sys-dba-password "<YOUR PASSWORD HERE>"
+                       :cmr-bootstrap-password "<YOUR PASSWORD HERE>"
+                       :cmr-ingest-password "<YOUR PASSWORD HERE>"
+                       :cmr-urs-password "<YOUR PASSWORD HERE>"}}}
+   ```
+
+2. `cmr setup dev`
```

Comment:
does this work, or will the markdown engine reset the count to 1 because of the spaces above? Have you viewed in an editor/viewer?